test/test_object.py

```diff
@@ -183,7 +183,7 @@ def foo(self):
         assert mod.f(PropAttr()) is True
         assert mod.f(PropAttrRaising()) is False
 
-    def test_setattr(self):
+    def test_setattr_delattr(self):
```

Comment:
I know the underlying implementation of delattr uses setattr, but could we keep the tests separate just so that each test function is a bit simpler?

Reply:
And the same comment for the other setattr / delattr tests.

Reply:
Okay, I can do that. I just merged them into one test because setattr/-item are obviously related to delattr/-item and in this way, we don't generate a second test module which is faster.
But I don't really mind. I'll separate it.